package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "devDependencies": {
     "babel-polyfill": "^6.20.0",
-    "firebase": "^3.6.3",
-    "firebase-tools": "^3.2.2",
+    "firebase": "5.5.8",
+    "firebase-tools": "^5.1.1",
```

Comment:
Were these upgrades intentional, can't see any mention in the commit message?

Reply:
As another comment, I wonder if we need the `firebase-tools` as a `devDependency` if it is installed by using `npm install -g` as in the README.md

Reply:
Yes intentional, the deploy command had stopped working, I will add to the commit message.  I think there are a lot of other dependancies that are out of date but I think I'll address those in a separate PR.